### ISSUE TITLE
Clicks limited to smaller region

### ISSUE THREAD

**Issue by daniel-lo** [Genymobile/scrcpy#3521]
Hi, first steps on my fire-tv-cube (all updates done). On the main screen I cannot access Settings and Profile .. so somewhat the space I can click in is limited. I also have problems navigating menus and would like to ask for the ideal command line for mouse control .. just in case somebody has already experimented this through.

Cheers

**Comment by daniel-lo** [comment]
@rom1v 

sorry .. too stupid to understand .. scrcpy windows launches with fire tv and I notice that click reagion is limited. How can I uuse adb shell in conjunction?

**Comment by rom1v** [comment]
From the scrcpy directory, click on `open_a_terminal_here.bat`, this opens a terminal. Execute `adb shell …` from there.

**Comment by rom1v** [comment]
If you want to know the coordinates easily, run `scrcpy -Vdebug` (when you click, it will print the coordinates in the console).

**Comment by rom1v** [comment]
> so the clicks are registed fine .. but nothing happens when I click on the cogwheel.

Even with `adb shell tap <x> <y>`? So the clicks are sent to your device, which does not react to them properly. Maybe there is a hidden overlay which capture them or something.

If your device is plugged over USB, can you try in [HID mode](https://github.com/Genymobile/scrcpy#physical-keyboard-simulation-hid): `scrcpy -KM` (you should see the mouse on your device).

**Comment by daniel-lo** [comment]
![grafik](https://user-images.githubusercontent.com/4311558/194275239-fee7a22e-3841-4394-b9b0-37613d08f143.png)

device is on my LAN

**Comment by daniel-lo** [comment]
And if tap would work .. it wouldn't explain why half the cogwheel is allocated to the apps tile.

**Comment by rom1v** [comment]
```
adb shell input tap 1764 581
```
(my fault, I forgot `input` in my last comment)

**Comment by daniel-lo** [comment]
ok .. still no reaction

**Comment by daniel-lo** [comment]
Hmm .. so I am the first FireTV Cube user of scrcpy?

**Comment by rom1v** [comment]
I don't know.

But you reproduce the problem without scrcpy (with just `adb shell input tap`), so it could not be a bug in scrcpy :)

**Comment by daniel-lo** [comment]
keyboard input works .. how do I simulate the burger key?

**Comment by karlic** [comment]
> Hmm .. so I am the first FireTV Cube user of scrcpy?

No. I also have this problem on TWO Fire TV Cubes. I'm connecting over TCP(?)/IP (NOT USB)
`adb disconnect; adb connect XX.XX.XX.XX:5555; scrcpy -b2M -m800 --max-fps 25`
If I use the mouse to click on any of the five leftmost icons, I get a response as if I had pressed the big round button on the remote (i.e., as expected). 
If I mouse click on the Netflix or other rectangles, the rectangle is selected as if I had moved the cursor using the remote and a second click opens the appropriate app, as if I had pressed the round button on the remote.
If I mouse click on the gear icon at the far right, nothing at all happens.
HOWEVER, if I use the keyboard arrow keys to navigate to the gear icon the settings are displayed and pressing return moves the cursor down to the settings squares.
When in the Netflix app, the mouse doesn't work on anything (that I can see).

The problem also occurs (probably unsurprisingly) when using scrcpy-mobile on iPadOS (see https://github.com/wsvn53/scrcpy-mobile/issues/13).

<img width="912" alt="Screenshot 2022-11-07 at 19 08 54" src="https://user-images.githubusercontent.com/6543702/200395611-399a1569-e8bb-4693-8b22-78677af1e8f4.png">
